spark/v3.3/spark/src/test/java/org/apache/iceberg/spark/source/TestIcebergSourceHiveTables.java

```diff
@@ -43,11 +43,10 @@ public static void start() {
 
   @After
   public void dropTable() throws IOException {
-    Table table = catalog.loadTable(currentIdentifier);
-    Path tablePath = new Path(table.location());
-    FileSystem fs = tablePath.getFileSystem(spark.sessionState().newHadoopConf());
-    fs.delete(tablePath, true);
-    catalog.dropTable(currentIdentifier, false);
+    if (!catalog.tableExists(currentIdentifier)) {
+      return;
+    }
+    dropTable(currentIdentifier);
```

Comment:
nit: whitespace missing after }

Reply:
seems like the newline is not added? Overall we encourage newline after any logical blocks like if, for, while, try, etc.